tasks/models.py

```diff
@@ -30,8 +32,9 @@ def __str__(self) -> str:
 
 
 class Comment(models.Model):
-    user_id = models.ForeignKey(User, on_delete=models.RESTRICT, related_name='comments')
-    task_id = models.ForeignKey(Task, on_delete=models.CASCADE)
+    user = models.ForeignKey(
+        User, on_delete=models.RESTRICT, related_name='comments')
+    task = models.ForeignKey(Task, on_delete=models.CASCADE)
```

Comment:
Just another naming or there is a specific reason for the name change?

Reply:
for every foreign key django already adding _id in the end, so better to remove it now

Reply:
plz add a " related_name='comments' " to the field - task
task = models.ForeignKey(Task, on_delete=models.CASCADE, related_name='comments')
@AvivLiberman

Reply:
Done